### ISSUE TITLE
Tooltip z-Index

### ISSUE THREAD

**Issue by christianblandford** [saadeghi/daisyui#245]
Hi all,

Is there a way to adjust the z-index of just the tooltip pop up? It's important that I keep the Z-index of the children of the <div class="tooltip"> 0, but I need a z-index of 1 on the tooltip text to avoid clipping under another element on the page.

I have tried setting Z-index for .tooltip, but this obviously changed the z-index of the children too. I'm sure I'm missing something obvious. I also tried .tooltip::before, but it did not seem to have any effect.

**Comment by christianblandford** [comment]
After further investigation, this seems to be caused by something unqiuue to my app, when I set the tooltip-open class, it works fine. So something must be going on with my hover actions.

**Comment by saadeghi** [comment]
If anyone looking for a way to target the tooltip text,  
1. you can use `.tooltip:before`: https://play.tailwindcss.com/BSJ2cgdfa3?file=css
2. or if you use JIT, you can use `before:` utility classes: https://play.tailwindcss.com/hrXJ1f0v2c

**Comment by christianblandford** [comment]
Thanks @saadeghi 

I discovered the issue was caused by a hover:brightness-125 class on a parent element. When this class was set, the tooltip clipped behind the other element. When it was not set, the tooltip displayed as normal. Have you ever seen anything like this?

![with_parent_hover_listener](https://user-images.githubusercontent.com/16197066/137181187-9b4ee46c-9cb6-4976-bf82-3700a6decc0c.png)
![without_parent_hover](https://user-images.githubusercontent.com/16197066/137181192-eff3699f-ab51-4a3e-aa31-11fb6cedfd89.png)
?
